samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/state_binding.c

```diff
@@ -305,4 +305,3 @@ void calculate_temp_target_values(u8_t type)
 			light_ctl_srv_user_data.delta_uv;
 	}
 }
-
```

Comment:
How is this change related to the commit?

Reply:
done !!